functions/nvm.fish

```diff
@@ -196,8 +196,8 @@ function _nvm_use
         end
     end
 
-    if set -l root (_nvm_find_up (pwd) $nvm_file)
-        echo $argv[1] >$root/$nvm_file
+    if set -l root (_nvm_find_up (pwd) $nvm_file); and test $ver != (cat $root/$nvm_file)
```

Comment:
i think you want to check not just `!=`, but that the contents of `$root/$nvm_file` don't *resolve* to `$ver`

Reply:
That makes sense, thanks 👍  Seems possible to extract the code at the top of `_nvm_use` to an `_nvm_resolve_version` function the can help.  I'll try that and see how it goes.

Reply:
I took a shot at it.  LMK if I'm on the right track.